uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -104,10 +104,16 @@ sub IsCandidateInfoValid {
     my $files_with_unmatched_patient_name = 0;
     my $is_candinfovalid                  = 0;
     my @row                               = ();
+
     ############################################################
+    ####Remove __MACOSX directory from the upload if found######
     ####Get a list of files from the folder#####################
     #############Loop through the files#########################
     ############################################################
+    my $cmd = "cd " . $this->{'uploaded_temp_folder'} . "; find -name '__MACOSX' | xargs rm -rf";
```

Comment:
change command to use delete option with `find` instead of piping into `xargs`

Reply:
@davidblader can you please re-test? I modified the one line as was discussed in the Imaging meeting of Nov'24th.

@driusan 
is this line better now?

Reply:
Is there something wrong with the `-delete` option?

@nicolasbrossard also pointed out that you don't need the cd since find takes a directory as the first parameter.

Reply:
@driusan @nicolasbrossard 
so I changed the command line again,
is this better?